workflow/controller/agent.go

```diff
@@ -40,21 +40,21 @@ func (woc *wfOperationCtx) reconcileAgentPod(ctx context.Context) error {
 	}
 	// Check Pod is just created
 	if pod.Status.Phase != "" {
-		woc.updateAgentPodStatus(ctx, pod)
+		woc.updateAgentPodStatus(pod)
```

Comment:
Why did we remove ctx?

Reply:
`ctx` here is nerver used in `updateAgentPodStatus` now because I use `markTaskSetNodesError` instead of `markWorkflowError` and `make pre-commit -B` will fail if I keep this unused `ctx`.